common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -79,6 +79,12 @@ class SequenceFields:  # lint-amnesty, pylint: disable=missing-class-docstring
         scope=Scope.settings,
     )
 
+    due_num_weeks = Integer(
+        display_name=_("Number of Weeks Due By"),
+        help=_("Enter the number of weeks the problems are due by"),
```

Comment:
Also while I'm thinking about this help text, might be good to also add a note in here about the waffle flag so people know it isn't always accessible. Perhaps a comment above the field too.